Services/MainMenu/classes/Provider/StandardTopItemsProvider.php

```diff
@@ -171,14 +174,20 @@ static function () : bool {
             ->withPosition(70)
             ->withVisibilityCallable($this->basic_access_helper->hasAdministrationAccess());
 
+        $dd_renderer = new TopParentItemDrilldownRenderer();
```

Comment:
This is because the administration menu should be the only-one which will be rendered as a Drilldown, right? I'm currently not sure if we should go this way OR whether we should introduce a dedicated `Item` in the Globalscreen-Scope `MainMenu`. Currently one type of `Item` results in one type of `UI\Component`. This is a predictable behaviour, every `Provider` which provides some type if `Items` will result in the same rendering of this type. One can replace the "renderer" – a.k.a "GS-item to UI-Component translator" – as a whole and all `Items` of the same type will be translated like this.

Both ways have their advantages and disadvantages and I'm currently not sure which way to go ;-)

Reply:
I'm not sure if adm will remain the only entry, but it certainly is a promising candidate :)
What I thought was, that with the DD for Administration, I'm actually not changing the nature of adm as topParentItem, but only display in a different way. 
The item-solution would be additional, right ? i.e. we would move the renderer from provider to collector?